src/action.c

```diff
@@ -1026,6 +1037,29 @@ actions_run(struct view *activator, struct server *server,
 				wlr_log(WLR_ERROR, "Invalid SnapToRegion id: '%s'", region_name);
 			}
 			break;
+		case ACTION_TYPE_CYCLE_IN_REGION:
+			{
+				if (!view) {
+					break;
+				}
+				struct output *output = view->output;
+				if (!output) {
+					break;
+				}
```

Comment:
Should likely be `output_is_usable(view->output)` rather than `!output` to catch leased / disabled outputs.

Reply:
Thanks, fixed.  Your comment is probably relevant to the function I copied it from too, but that'd be for another patch.